### DISCUSSION TITLE
QField cloud server down?

### DISCUSSION THREAD

**Discussion by BoswachterMarc** [opengisch/QField#5738] [Q&A]
I was able to push changes but syncing is not working. Downloading a project isn't possible neither. Perhaps there is a temporary problem on the server side?

![afbeelding](https://github.com/user-attachments/assets/edb0970a-2949-4117-b325-c14af69a02c5)

![afbeelding](https://github.com/user-attachments/assets/0cbe28b7-45ba-4c1f-8a96-5ce97f8363a8)

**Comment by suricactus** [comment]
Thanks for the report. I will have a look on your specific project.

As for the red line on qfield.cloud - this is for the static website that is at https://qfield.cloud . The checkly.com service we are using for automated tests seems to have changed their framwork version and the code we had for checking was wrong. Now I have fixed it and all should be green there.

**Comment by suricactus** [comment]
Just checked your issue. It seems there was another temporary issue with our fille storage provider. Can you retry the package job? I am pretty sure all should be good now.

**Comment by BoswachterMarc** [comment]
Oh, it still doesn't work. I am also trying to download a different, small community project on a new mobile device and that also gives a download error. In both cases, the data storage is within the limit. I'm going to sleep now, tomorrow is another day ;-)

![afbeelding](https://github.com/user-attachments/assets/81170945-e924-429b-8515-56eee51ddd19)

**Comment by BoswachterMarc** [comment]
Just tried again, this time it worked for both projects. Great ;-)
